src/components/proposalBooking/CreateUpdateVisit.tsx

```diff
@@ -112,7 +112,9 @@ function CreateUpdateVisit({ event, close }: CreateUpdateVisitProps) {
             <Button
               disabled={isSubmitting}
               type="submit"
-              data-cy="create-visit-button"
+              variant="contained"
```

Comment:
Probably variant and color are not needed here because we are setting those as defaults in the theme.ts